Week-2/Assignment 2.2/index.css

```diff
@@ -0,0 +1,80 @@
+#main {
+  width: 200px;
+  height: 200px;
+  top: 50%;
+  left: 50%;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+h1 {
+  position: absolute;
+  align-self: center;
+}
+
+#A {
+  background-color: red;
+  height: 100px;
+  width: 100px;
+  position: relative;
+  top: 0px;
+  left: 0px;
+  transform-origin: top left;
+  transition: transform 1s ease;
+}
+#B {
+  background-color: yellow;
+  height: 100px;
+  width: 100px;
+  position: relative;
+  top: 0px;
+  left: 0px;
+  transform-origin: top right;
+  transition: transform 1s ease;
+}
+#C {
```

Comment:
PLease read this http://bdavidxyz.com/blog/how-to-name-css-classes/  -> how to name the class and its best practises